docs/project-list.md

```diff
@@ -0,0 +1,103 @@
+---
+date-generated: 2020-03-24T14:44:10
+tool: NuGetTasks.GenerateMarkdownDoc
+---
+
+
+# NuGet Project Overview
+
+Below is a list of all source code projects for NuGet libraries and supported NuGet clients
+
+
+
+## Projects
+
+All shipped NuGet libraries and clients lives in `src/` folder.
+
+Projects count: 37
+
+- [`src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj`](../src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.csproj): NuGet Command Line Interface.
```

Comment:
How about we use the project name as the link name? 
Right now it looks kind of clunky.

Reply:
Changed to project name.